ios/FluentUI/Navigation/Shy Header/ShyHeaderView.swift

```diff
@@ -55,16 +55,16 @@ class ShyHeaderView: UIView, TokenizedControlInternal {
     }
 
     private struct Constants {
-        static let contentHorizontalPadding: CGFloat = 16
-        static let contentTopPadding: CGFloat = 6
-        static let contentTopPaddingCompact: CGFloat = 10
-        static let contentTopPaddingCompactForLargePhone: CGFloat = 0
-        static let contentBottomPadding: CGFloat = 10
-        static let contentBottomPaddingCompact: CGFloat = 6
+        static let contentHorizontalPadding: CGFloat = GlobalTokens.spacing(.size160)
+        static let contentTopPadding: CGFloat = GlobalTokens.spacing(.size80)
+        static let contentTopPaddingCompact: CGFloat = GlobalTokens.spacing(.size80)
```

Comment:
Since `contentTopPadding`, `contentTopPaddingCompact`, `contentBottomPadding`, and `contentBottomPaddingCompact` are all the same, could we simplify them to just `static let contentVerticalPadding: CGFloat = GlobalToken.spacing(.size80)`? I know `contentTopPaddingCompactForLargePhone` isn't the same, but reducing the other four should help.

Reply:
Honestly, I don't think distinct `...CompactForLargePhone` constants have any business existing. Whether we use a `...Compact` constant or `...CompactForLargePhone` constant is determined by the *horizontal* size class, but the constants here talk about *vertical* spacing.

That being said, the `...CompactForLargePhone` constants (like on the Pro Max in the following screenshot) result in a more vertically centered title, so we might want to ultimately use those values. I'll play around with these some more and see what I can come up with.

<img width="660" alt="Screenshot 2023-04-21 at 11 26 00 AM" src="https://user-images.githubusercontent.com/717674/233718520-10f84796-f449-4854-b162-023269099544.png">

Reply:
The course of action here will depend on how we handle [this comment](https://github.com/microsoft/fluentui-apple/pull/1713#discussion_r1173267910), so I'm just gonna resolve this one to keep all the discussion in one place.